README.md

```diff
@@ -1,16 +1,14 @@
-Logger - Logging singleton for Godot Engine
+Godot Logger - Logging Addon for Godot Engine
 ===========================================
 
-The *Logger* class is a GDScript singleton that provides a logging API for
+The *Godot Logger* is a plugin that provides a logging API for
 projects developed with [Godot Engine](https://godotengine.org).
 
 # Usage
 
-Copy the `logger.gd` file into your project folder, and define it as an autoloaded
-singleton in your project's settings (e.g. with the name *Logger*):
-```
-Logger="*res://logger.gd"
-```
+1. Clone this repository to `godot_project/addons/`.
```

Comment:
Experience showed me that many users, including developers, don't understand this kind of pseudo path and will take it literally (and then wonder why it doesn't work).

So I would do:
```suggestion
1. Clone or extract this repository as the `res://addons/godot-logger` folder in your project.
```

Reply:
Done  👍